src/openscad.cc

```diff
@@ -783,12 +783,42 @@ int gui(const vector<string> &inputFiles, const fs::path &original_path, int arg
 }
 #endif // OPENSCAD_QTGUI
 
+#include <execinfo.h>
```

Comment:
What's that include used for? This breaks Windows (MXE based) compilation. Can we make it conditional maybe?